src/Liquidator.sol

```diff
@@ -161,10 +171,10 @@ contract Liquidator is Owned {
      */
     function setAuctionCurveParameters(uint16 halfLifeTime, uint16 cutoffTime_) external onlyOwner {
         //Checks that new parameters are within reasonable boundaries.
-        if (halfLifeTime <= 120) revert Liquidator_HalfLifeTimeTooLow(); // 2 minutes
-        if (halfLifeTime >= 28_800) revert Liquidator_HalfLifeTimeTooHigh(); // 8 hours
-        if (cutoffTime_ <= 3600) revert Liquidator_CutOffTooLow(); // 1 hour
-        if (cutoffTime_ >= 64_800) revert Liquidator_CutOffTooHigh(); // 18 hours
+        require(halfLifeTime > 120, "LQ_SACP: halfLifeTime too low"); // 2 minutes
```

Comment:
Why are the custom errors removed?

Reply:
It got squished in one of the merges, it is reverted now